enterprise-metrics/main.libsonnet

```diff
@@ -184,7 +185,6 @@ local removeNamespaceReferences(args) = std.map(function(arg) std.strReplace(arg
       + container.withPorts(cortex.util.defaultPorts)
       + container.resources.withLimits({ cpu: '2', memory: '4Gi' })
       + container.resources.withRequests({ cpu: '500m', memory: '1Gi' })
-      + cortex.jaeger_mixin
```

Comment:
what's the purpose of this removal? seems tangential to the PR but I don't know much about it.

Reply:
Yeah, sorry for the confusion, I forgot I included the removal in this PR. 
```
refactor: Remove empty jaeger_mixins 

Since no jaeger_agent_host is specified in the Cortex configuration,
these are empty mixins.
```

Reply:
In future, I will make sure that this is part of a separate PR, sometimes I get a bit caught up in cleaning up everything I can :D